src/renderer/components/layout/tab-layout.tsx

```diff
@@ -0,0 +1,44 @@
+import "./tab-layout.scss";
+
+import React, { ReactNode } from "react";
+import { matchPath, RouteProps } from "react-router-dom";
+import { cssNames } from "../../utils";
+import { Tab, Tabs } from "../tabs";
+import { ErrorBoundary } from "../error-boundary";
+import { navigate, navigation } from "../../navigation";
+import { getHostedCluster } from "../../../common/cluster-store";
+
+export interface TabRoute extends RouteProps {
+  title: React.ReactNode;
+  url: string;
+}
+
+interface Props {
+  children: ReactNode;
+  className?: any;
+  tabs?: TabRoute[];
+  contentClass?: string;
+}
+
+export const TabLayout = ({ className, contentClass, tabs, children }: Props) => {
```

Comment:
This component should be wrapped into `observer` decorator from `mobx-react` in order to react to observables like `getHostedCluster()` and `navigation.location.pathname`

Reply:
@ixrock Thanks for your review. Wrapped `TabLayout` with `observer` in the latest commit.